css/css-box/margin-trim/block-container-non-adjoining-item-ref.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>CSS margin-trim: block-container-non-adjoining-item</title>
+<link rel="author" title="Sammy Gill" href="mailto:sammy.gill@apple.com">
+<link rel="help" href="https://drafts.csswg.org/css-box-4/#margin-trim-block">
+<meta name="assert" content="non-adjoining first child should still have its margins trimmed and the container should keep its set margin">
+<style>
+.container {
+    background-color: green;
+    width: 100px;
+    margin-bottom: 100px;
+    padding-bottom: 50px;
+    background-clip: padding-box;
+}
+.outer {
+    background-color: green;
+    width: 100px;
+    height: 50px;
+}
+.child {
+    height: 50px;
+}
+</style>
+</head>
+<body>
+<div>
+<div class="container">
+    <div class="child"></div>
+</div>
+<div>This text should be 100px below the green square.</div>
+</div>
+</body>
+</html>
```

Comment:
nit: this and several other files are missing a newline at the end (shown as a red warning-sign in some github diff viewer UIs, and shown as `\ No newline at end of file` at https://github.com/web-platform-tests/wpt/commit/de3b945add7f92fbb0037eeab798116e21f16775.diff )

Ideally all of these files should have a trailing newline to avoid those warnings.